http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/RuntimeServiceInterp.scala

```diff
@@ -1014,12 +1022,13 @@ object RuntimeServiceInterp {
                                                    autoFreezeConfig: AutoFreezeConfig): Int =
     autopause match {
       case None =>
-        autoFreezeConfig.autoFreezeAfter.toMinutes.toInt
+        autoPauseOffValue
       case Some(false) =>
         autoPauseOffValue
       case _ =>
-        if (autopauseThreshold.isEmpty) autoFreezeConfig.autoFreezeAfter.toMinutes.toInt
-        else Math.max(autoPauseOffValue, autopauseThreshold.get)
+        if (autopauseThreshold.isEmpty) 30 //TODO what behavior do we want when enabled but no threshold
```

Comment:
Magic number alert :) If we default to a value (like `30` here), it should probably come from a config, or at the least it should be defined somewhere higher.

Reply:
Planning to change! Talked to Qi about it earlier